content/project/hal.md

```diff
@@ -9,14 +9,18 @@ main_site_item_images: ["1.jpg", "2.jpg", "3.jpg", "4.jpg"]
 start_date:
 end_date:
 short_description:
-    "Łazik marsjański „HAL-062” w tym roku brał udział w University Rover
-    Challenge. Są to odbywających się w USA zawody, na których ma być
-    symulowana współpraca pomiędzy robotem a astronautą podczas misji na
-    marsie. Są to najbardziej prestiżowe i rozpoznawalne zawody robotyczne na
-    świecie. Robot jest wyposażony w zawieszenie typu Rocker - Bogie, napęd
-    różnicowy oraz manipulator o 6 stopniach swobody. Dodatkowo posiada moduł
-    z kamerami oraz GPS-em, używany do autonomicznej nawigacji podczas jednej
-    z konkurencji. Aktualnie jego następna, ulepszona wersja jest
-    przygotowywana na kolejną edycję konkursu, która odbędzie się w maju
-    przyszłego roku."
+    Łazik marsjański "HAL-062" jest naszym największym projektem. 
```

Comment:
Nie sądzisz, że to stwierdzenie, jest trochę na wyrost i faworyzuje ten, jeden projekt?